config/application.rb

```diff
@@ -22,7 +22,7 @@
 module InfoFrontend
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
-    config.load_defaults 6.0
+    config.load_defaults 6.1
```

Comment:
One question: does setting `load_defaults` here to 6.1 bypass the new framework defaults file (where they are all commented out) and apply them anyway behind the scenes?

Reply:
Yes, this change applies all the new defaults. I edited the commit to clarify this.

Reply:
Thanks for the update - read through and makes sense to me!